### PR TITLE
SAK-33732 - Error in time while modifying an event with timeslots

### PR DESCRIPTION
**Pull request by jesusmmp**

### CODE REVIEW

File: signup/tool/src/java/org/sakaiproject/signup/tool/jsf/organizer/UserDefineTimeslotBean.java [hunk 1]
```diff
@@ -411,10 +421,11 @@ public void validateTimeslots(ActionEvent e) {
 				tsWrp.setErrorStyle(this.errorStyleValue);
 				Utilities.addErrorMessage(MessageFormat.format(Utilities.rb
 						.getString("event.endTimeslot_should_after_startTimeslot"),
-						new Object[] { position }));
+						new Object[] { i }));
```
Comment (ern) [inline signup/tool/src/java/org/sakaiproject/signup/tool/jsf/organizer/UserDefineTimeslotBean.java]: Explicit creation of an array is not needed when using varargs.

File: signup/tool/src/java/org/sakaiproject/signup/tool/jsf/organizer/UserDefineTimeslotBean.java [hunk 2]
```diff
@@ -386,21 +389,28 @@ private boolean isEmptyTimeslotWrpList(){
...
 		for (TimeslotWrapper tsWrp : this.timeSlotWrpList) {
 			if(tsWrp.getDeleted())
 				continue;//skip
 			
-			Map<String, String> params = FacesContext.getCurrentInstance().getExternalContext().getRequestParameterMap();
-
 			String isoStartTime = params.get((position - 1) + HIDDEN_ISO_STARTTIME);
+			String isoEndTime = params.get((position - 1) + HIDDEN_ISO_ENDTIME);
+			
+			if (isoStartTime == null && isoEndTime == null) {
```
Comment (ern) [inline signup/tool/src/java/org/sakaiproject/signup/tool/jsf/organizer/UserDefineTimeslotBean.java]: This method is about validating parameters, is just skipping parameters really accurate? Maybe the location where the parameters are added to the request is actually wrong and this method is correctly identifying that?

Comment (jesusmmp) [reply signup/tool/src/java/org/sakaiproject/signup/tool/jsf/organizer/UserDefineTimeslotBean.java]: You have to find the next one isoStartTime and isoEndTime params.
These params don't depend on the position of the Timeslot.
I attach an image: https://snag.gy/tW1V4e.jpg

**Comment by mpellicer** [comment]
Tested successfully in the Atica server (S2U QA server)

**Pull request merged**